shopping_mall/backend/app/routers/chatbot.py

```
@@ -64,9 +114,223 @@ def rate_chat_log(log_id: int, body: ChatRating, db: Session = Depends(get_db)):
+            session_dict.message_preview = last_log.answer
+
+        results.append(session_dict)
+
+    return results
+
+
+@router.get("/sessions/active", response_model=Optional[ChatSessionResponse])
+def get_active_session(
+    user_id: int = Query(...),
+    authenticated_user_id: int = Depends(_get_user_id),
+    db: Session = Depends(get_db),
```

Ruff (0.15.9)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)